electron/renderer/src/components/Webviews.js

```diff
@@ -59,13 +59,17 @@ class Webviews extends Component {
     );
   }
 
-  _getEnvironmentUrl(account) {
+  _getEnvironmentUrl(account, forceLogin) {
     const envParam = decodeURIComponent(new URL(window.location).searchParams.get('env'));
     const url = new URL(envParam);
 
     // pass account id to webview so we can access it in the preload script
     url.searchParams.set('id', account.id);
 
+    if (forceLogin) {
+      url.hash = '#login';
```

Comment:
Let's try to not go this path again. The dependency between browser and webapp should be as limited as possible. We already have outdated wrapper versions that contained this at some point 🙈